python/cuml/test/dask/test_linear_regression.py

```diff
@@ -42,6 +42,7 @@ def load_data(nrows, ncols, cached='data/mortgage.npy.gz'):
     else:
         print('use random data')
         X = np.random.rand(nrows, ncols)
+        y = np.random.rand(nrows, 1)
```

Comment:
how did this work before? Is it even getting called?

Reply:
#1001

Reply:
I believe the linear regression tests were being skipped in dask-cuml. @dantegd can correct me if I'm wrong, but I think the reasoning was because of the dependency on libcumlmg during 0.6 release.

Reply:
Actually, it looks like the linear regression tests are still being skipped. Should we enable these now? 

https://github.com/rapidsai/cuml/blob/0b861d9c32c0b902b1aa95be4c96b179795bc95a/python/cuml/test/dask/test_linear_regression.py#L53